It seems so, and the errors is due to the change of the currency, which I think you have sent a PR to Odoo, isn't it?

`purchase`
<details>
2018-04-04 10:34:04,598 23268 [1;34m[1;49mDEBUG[0m openerp_test openerp.tools.yaml_import: Exception during evaluation of !python block in yaml_file /home/travis/OCB-9.0/addons/purchase/test/fifo_price.yml.
Traceback (most recent call last):
File "/home/travis/OCB-9.0/openerp/tools/yaml_import.py", line 617, in process_python
unsafe_eval(code_obj, {'ref': self.get_id}, code_context)
File "/home/travis/OCB-9.0/addons/purchase/test/fifo_price.yml", line 6, in <module>
company.currency_id = company.env['res.currency'].browse(ref('base.EUR'))
File "/home/travis/OCB-9.0/openerp/fields.py", line 867, in __set__
record.write({self.name: self.convert_to_write(value)})
File "/home/travis/OCB-9.0/openerp/api.py", line 248, in wrapper
return new_api(self, *args, **kwargs)
File "/home/travis/OCB-9.0/addons/l10n_fr_certification/models/res_company.py", line 27, in write
res = super(ResCompany, self).write(vals)
File "/home/travis/OCB-9.0/openerp/api.py", line 248, in wrapper
return new_api(self, *args, **kwargs)
File "/home/travis/OCB-9.0/addons/account/models/company.py", line 118, in write
raise UserError(_('You cannot change the currency of the company since some journal items already exist'))
UserError: ('You cannot change the currency of the company since some journal items already exist', None)
2018-04-04 10:34:04,601 23268 [1;31m[1;49mERROR[0m openerp_test openerp.tools.yaml_import: ('You cannot change the currency of the company since some journal items already exist', None)
Traceback (most recent call last):
File "/home/travis/OCB-9.0/openerp/tools/yaml_import.py", line 884, in process
self._process_node(node)
File "/home/travis/OCB-9.0/openerp/tools/yaml_import.py", line 897, in _process_node
self.process_python(node)
File "/home/travis/OCB-9.0/openerp/tools/yaml_import.py", line 617, in process_python
unsafe_eval(code_obj, {'ref': self.get_id}, code_context)
File "/home/travis/OCB-9.0/addons/purchase/test/fifo_price.yml", line 6, in <module>
company.currency_id = company.env['res.currency'].browse(ref('base.EUR'))
File "/home/travis/OCB-9.0/openerp/fields.py", line 867, in __set__
record.write({self.name: self.convert_to_write(value)})
File "/home/travis/OCB-9.0/openerp/api.py", line 248, in wrapper
return new_api(self, *args, **kwargs)
File "/home/travis/OCB-9.0/addons/l10n_fr_certification/models/res_company.py", line 27, in write
res = super(ResCompany, self).write(vals)
File "/home/travis/OCB-9.0/openerp/api.py", line 248, in wrapper
return new_api(self, *args, **kwargs)
File "/home/travis/OCB-9.0/addons/account/models/company.py", line 118, in write
raise UserError(_('You cannot change the currency of the company since some journal items already exist'))
UserError: ('You cannot change the currency of the company since some journal items already exist', None)
2018-04-04 10:34:04,601 23268 [1;31m[1;49mERROR[0m openerp_test openerp.tests: module purchase: an exception occurred in a test
Traceback (most recent call last):
File "/home/travis/OCB-9.0/openerp/modules/loading.py", line 44, in load_test
_load_data(cr, module_name, idref, mode, 'test')
File "/home/travis/OCB-9.0/openerp/modules/loading.py", line 98, in _load_data
tools.convert_file(cr, module_name, filename, idref, mode, noupdate, kind, report)
File "/home/travis/OCB-9.0/openerp/tools/convert.py", line 847, in convert_file
convert_yaml_import(cr, module, fp, kind, idref, mode, noupdate, report)
File "/home/travis/OCB-9.0/openerp/tools/yaml_import.py", line 948, in yaml_import
yaml_interpreter.process(yaml_string)
File "/home/travis/OCB-9.0/openerp/tools/yaml_import.py", line 884, in process
self._process_node(node)
File "/home/travis/OCB-9.0/openerp/tools/yaml_import.py", line 897, in _process_node
self.process_python(node)
File "/home/travis/OCB-9.0/openerp/tools/yaml_import.py", line 617, in process_python
unsafe_eval(code_obj, {'ref': self.get_id}, code_context)
File "/home/travis/OCB-9.0/addons/purchase/test/fifo_price.yml", line 6, in <module>
company.currency_id = company.env['res.currency'].browse(ref('base.EUR'))
File "/home/travis/OCB-9.0/openerp/fields.py", line 867, in __set__
record.write({self.name: self.convert_to_write(value)})
File "/home/travis/OCB-9.0/openerp/api.py", line 248, in wrapper
return new_api(self, *args, **kwargs)
File "/home/travis/OCB-9.0/addons/l10n_fr_certification/models/res_company.py", line 27, in write
res = super(ResCompany, self).write(vals)
File "/home/travis/OCB-9.0/openerp/api.py", line 248, in wrapper
return new_api(self, *args, **kwargs)
File "/home/travis/OCB-9.0/addons/account/models/company.py", line 118, in write
raise UserError(_('You cannot change the currency of the company since some journal items already exist'))
UserError: ('You cannot change the currency of the company since some journal items already exist', None)
</details>

`stock_dropshipping`
<details>
2018-04-04 10:38:28,416 23268 [1;34m[1;49mDEBUG[0m openerp_test openerp.tools.yaml_import: Exception during evaluation of !python block in yaml_file /home/travis/OCB-9.0/addons/stock_dropshipping/test/lifo_price.yml.
Traceback (most recent call last):
File "/home/travis/OCB-9.0/openerp/tools/yaml_import.py", line 617, in process_python
unsafe_eval(code_obj, {'ref': self.get_id}, code_context)
File "/home/travis/OCB-9.0/addons/stock_dropshipping/test/lifo_price.yml", line 5, in <module>
self.write(cr, uid, [ref('base.main_company')], {'currency_id':ref('base.EUR')}, context=context)
File "/home/travis/OCB-9.0/openerp/api.py", line 250, in wrapper
return old_api(self, *args, **kwargs)
File "/home/travis/OCB-9.0/openerp/api.py", line 381, in old_api
result = method(recs, *args, **kwargs)
File "/home/travis/OCB-9.0/addons/l10n_fr_sale_closing/models/res_company.py", line 23, in write
res = super(ResCompany, self).write(vals)
File "/home/travis/OCB-9.0/openerp/api.py", line 248, in wrapper
return new_api(self, *args, **kwargs)
File "/home/travis/OCB-9.0/addons/l10n_fr_certification/models/res_company.py", line 27, in write
res = super(ResCompany, self).write(vals)
File "/home/travis/OCB-9.0/openerp/api.py", line 248, in wrapper
return new_api(self, *args, **kwargs)
File "/home/travis/OCB-9.0/addons/account/models/company.py", line 118, in write
raise UserError(_('You cannot change the currency of the company since some journal items already exist'))
UserError: ('You cannot change the currency of the company since some journal items already exist', None)
2018-04-04 10:38:28,417 23268 [1;31m[1;49mERROR[0m openerp_test openerp.tools.yaml_import: ('You cannot change the currency of the company since some journal items already exist', None)
Traceback (most recent call last):
File "/home/travis/OCB-9.0/openerp/tools/yaml_import.py", line 884, in process
self._process_node(node)
File "/home/travis/OCB-9.0/openerp/tools/yaml_import.py", line 897, in _process_node
self.process_python(node)
File "/home/travis/OCB-9.0/openerp/tools/yaml_import.py", line 617, in process_python
unsafe_eval(code_obj, {'ref': self.get_id}, code_context)
File "/home/travis/OCB-9.0/addons/stock_dropshipping/test/lifo_price.yml", line 5, in <module>
self.write(cr, uid, [ref('base.main_company')], {'currency_id':ref('base.EUR')}, context=context)
File "/home/travis/OCB-9.0/openerp/api.py", line 250, in wrapper
return old_api(self, *args, **kwargs)
File "/home/travis/OCB-9.0/openerp/api.py", line 381, in old_api
result = method(recs, *args, **kwargs)
File "/home/travis/OCB-9.0/addons/l10n_fr_sale_closing/models/res_company.py", line 23, in write
res = super(ResCompany, self).write(vals)
File "/home/travis/OCB-9.0/openerp/api.py", line 248, in wrapper
return new_api(self, *args, **kwargs)
File "/home/travis/OCB-9.0/addons/l10n_fr_certification/models/res_company.py", line 27, in write
res = super(ResCompany, self).write(vals)
File "/home/travis/OCB-9.0/openerp/api.py", line 248, in wrapper
return new_api(self, *args, **kwargs)
File "/home/travis/OCB-9.0/addons/account/models/company.py", line 118, in write
raise UserError(_('You cannot change the currency of the company since some journal items already exist'))
UserError: ('You cannot change the currency of the company since some journal items already exist', None)
2018-04-04 10:38:28,417 23268 [1;31m[1;49mERROR[0m openerp_test openerp.tests: module stock_dropshipping: an exception occurred in a test
Traceback (most recent call last):
File "/home/travis/OCB-9.0/openerp/modules/loading.py", line 44, in load_test
_load_data(cr, module_name, idref, mode, 'test')
File "/home/travis/OCB-9.0/openerp/modules/loading.py", line 98, in _load_data
tools.convert_file(cr, module_name, filename, idref, mode, noupdate, kind, report)
File "/home/travis/OCB-9.0/openerp/tools/convert.py", line 847, in convert_file
convert_yaml_import(cr, module, fp, kind, idref, mode, noupdate, report)
File "/home/travis/OCB-9.0/openerp/tools/yaml_import.py", line 948, in yaml_import
yaml_interpreter.process(yaml_string)
File "/home/travis/OCB-9.0/openerp/tools/yaml_import.py", line 884, in process
self._process_node(node)
File "/home/travis/OCB-9.0/openerp/tools/yaml_import.py", line 897, in _process_node
self.process_python(node)
File "/home/travis/OCB-9.0/openerp/tools/yaml_import.py", line 617, in process_python
unsafe_eval(code_obj, {'ref': self.get_id}, code_context)
File "/home/travis/OCB-9.0/addons/stock_dropshipping/test/lifo_price.yml", line 5, in <module>
self.write(cr, uid, [ref('base.main_company')], {'currency_id':ref('base.EUR')}, context=context)
File "/home/travis/OCB-9.0/openerp/api.py", line 250, in wrapper
return old_api(self, *args, **kwargs)
File "/home/travis/OCB-9.0/openerp/api.py", line 381, in old_api
result = method(recs, *args, **kwargs)
File "/home/travis/OCB-9.0/addons/l10n_fr_sale_closing/models/res_company.py", line 23, in write
res = super(ResCompany, self).write(vals)
File "/home/travis/OCB-9.0/openerp/api.py", line 248, in wrapper
return new_api(self, *args, **kwargs)
File "/home/travis/OCB-9.0/addons/l10n_fr_certification/models/res_company.py", line 27, in write
res = super(ResCompany, self).write(vals)
File "/home/travis/OCB-9.0/openerp/api.py", line 248, in wrapper
return new_api(self, *args, **kwargs)
File "/home/travis/OCB-9.0/addons/account/models/company.py", line 118, in write
raise UserError(_('You cannot change the currency of the company since some journal items already exist'))
UserError: ('You cannot change the currency of the company since some journal items already exist', None)
</details>

--- a/.travis.yml
+++ b/.travis.yml
@@ -19,7 +19,7 @@ python:
   - "2.7"
 
 env:
-  - VERSION="9.0" ODOO_REPO="OCA/OCB" LINT_CHECK="0" EXTRA_EXCLUDE="theme_bootswatch"
+  - VERSION="9.0" ODOO_REPO="OCA/OCB" LINT_CHECK="0" WEBSITE_REPO="1"
 
 virtualenv:
   system_site_packages: true
@@ -31,13 +31,10 @@ before_install:
 install:
   - ln -s ${HOME}/build/${ODOO_REPO} ${HOME}/${ODOO_REPO#*/}-${VERSION}
   - git clone --depth=1 https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
-  - pip install -r ${HOME}/maintainer-quality-tools/travis/requirements.txt
-  - pip install -q QUnitSuite flake8 coveralls pylint
-  - ln -s `which nodejs` $HOME/maintainer-quality-tools/travis/node
-  - npm install less less-plugin-clean-css
   - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
+  - travis_install_nightly
   - export INCLUDE=$(getaddons.py -m ${HOME}/build/${ODOO_REPO}/openerp/addons ${HOME}/build/${ODOO_REPO}/addons)
-  - export EXCLUDE=hw_scanner,hw_escpos,$EXTRA_EXCLUDE
+  - export EXCLUDE=hw_scanner,hw_escpos,theme_bootswatch
   - cp ${HOME}/maintainer-quality-tools/cfg/.coveragerc .
 
 script:

